apps/studio/pages/project/[ref]/logs/edge-logs.tsx

```diff
@@ -19,6 +19,6 @@ export const LogPage: NextPageWithLayout = () => {
   )
 }
 
-LogPage.getLayout = (page) => <LogsLayout title="Edge Logs">{page}</LogsLayout>
+LogPage.getLayout = (page) => <LogsLayout title="API Gateway">{page}</LogsLayout>
```

Comment:
we renamed it to API Gateway in the sidebar a while back but the title was still showing the old name